docs/diagrams/StorageLoadSequence.puml

```diff
@@ -0,0 +1,88 @@
+@startuml
+!include style.puml
+
+Box
+    participant Main UI_COLOR
+    participant ":Storage" as Storage STORAGE_COLOR
+    participant ":Family" as Family MODEL_COLOR
+    participant ":PersonList" as PersonList MODEL_COLOR
+    participant ":File" as File STORAGE_COLOR_T2
+end box
+
+-> Main: initialisePlanitarium()
+activate Main
+create Storage
+Main -> Storage: new()
+activate Storage
+Storage --> Main: :Storage
+deactivate Storage
+Main -> Storage: loadData()
+activate Storage
+Storage -> Storage : checkFileExists
+activate Storage
+opt fileDoesNotExist
+    create File
+    Storage -> File : createNewFile()
+    activate File
+    File --> Storage
+    deactivate File
+end
+Storage --> Storage
+deactivate Storage
+create Family
+Storage -> Family: new()
+activate Family
+create PersonList
+loop for each group
+Family -> PersonList : new()
+activate PersonList
+PersonList --> Family : :PersonList
+deactivate PersonList
+end
+Family --> Storage : :Family
+deactivate Family
+Storage -> Storage: readSaveFile()
+activate Storage
+    loop until all lines read
+    Storage -> Storage : addCorrespondingInfo()
+    activate Storage
+        alt user
+            Storage -> Family : addPerson(groupID:int, name:String)
+            activate Family
+            Family -> PersonList: addPerson(name:String)
+            activate PersonList
+            PersonList --> Family
+            deactivate PersonList
+            Family --> Storage
+            deactivate Family
+        else #PINK expenditure
+            Storage -> Family : addExpend(groupID:int, personID:int, description:String, amount:double, categoryID:int, isPermanent:Boolean)
+            activate Family
+            Family -> PersonList : addExpend(personID:int, description:String, amount:double, categoryID:int, isPermanent:Boolean)
+            activate PersonList
+            PersonList --> Family
+            deactivate PersonList
+            Family --> Storage
+            deactivate Family
+        else #GREEN income
+            Storage -> Family : addIncome(groupID:int, personID:int, description:String, amount:double, isPermanent:Boolean)
+            activate Family
+            Family -> PersonList : addIncome(personID:int, description:String, amount:double, isPermanent:Boolean)
+            activate PersonList
+            PersonList --> Family
+            deactivate PersonList
+            Family --> Storage
+            deactivate Family
```

Comment:
Is there a reason why the boxes are pink and green here? The green background might make the text in black hard to read.

Reply:
In addition, again, consider not writing the full function call so the graph is less stretched horizontally and more readable?